app/serializers/membership_serializer.rb

```diff
@@ -4,4 +4,9 @@ class MembershipSerializer < ActiveModel::Serializer
   has_one :group, serializer: GroupSerializer, root: 'groups'
   has_one :user, serializer: UserSerializer, root: 'users'
   has_one :inviter, serializer: UserSerializer, root: 'users'
+
+  def include_inviter?
+    [nil, true].include? (scope || {})[:include_inviter]
```

Comment:
This is a little trick which means that we include the inviter if there is no scope (most of the time), or if we explicitly set 'include_inviter' to true, but not if we set `include_inviter` to be explicitly false

IE:
```
MembershipSerializer.new(membership, include_inviter: false).as_json
```

won't have the inviter included (we were having the inviter come back as an invitable user before, because it was included in the `user` query results)